sda-commons-dependency-check/build.gradle

```diff
@@ -35,3 +39,18 @@ dependencies {
   compile project(':sda-commons-shared-tracing')
   compile project(':sda-commons-shared-yaml')
 }
+
+/**
+ * A rule that forces all bouncycastle modules to use the same version.
+ */
+class BouncycastleAlignmentRule implements ComponentMetadataRule {
+  @Override
+  void execute(ComponentMetadataContext ctx) {
+    ctx.details.with {
+      if (id.group == "org.bouncycastle") {
+        // declare that the modules belong to the smallrye-config virtual platform
```

Comment:
I guess I know where you copied that from :wink

Reply:
😮